library/ws-widget/collection/src/lib/card-hubs-list/card-hubs-list.component.html

```diff
@@ -207,7 +207,7 @@ <h4 class="mat-h4 margin-bottom-remove nav_sub_heading">Coming soon</h4>
 
 <ng-template #userIndividualNew let-hubs="hubs">
   <ng-container *ngIf="hubs.hubname == 'Discuss' && !hubs.active">
-    <a [id]="hubs.hubname" class="mobile flex flex-wrap mb-4" (click)="toggleVisibility(); navigate()" role="menuitem" i18n-aria-label>
+    <a [id]="hubs.hubname" class="mobile flex flex-wrap mb-4" (click)="toggleVisibility(); navigate(); trackTelemetry(hubs.hubname)" role="menuitem" i18n-aria-label>
```

Comment:
Remove semicolon else build will fail.

Reply:
I've executed the prod build command locally without facing any problems, and we can use a semi-colon to separate function calls when necessary for multiple calls.